src/rest_framework_dso/renderers.py

```diff
@@ -246,13 +247,21 @@ def tune_serializer(self, serializer: Serializer):
 
         serializer.fields = csv_fields
 
-    def _get_csv_header(self, serializer: Serializer):
-        """Build the CSV header, including fields from sub-resources."""
+    def _get_csv_header(self, serializer: Serializer, request: HttpRequest | None = None):
+        """Build the CSV header, including fields from sub-resources.
+
+        If a _fields parameter is present on the request and it doesn't exclude a field,
+        we use that as the header, but still loop over the serializer fields to get the
+        labels.
+        """
         header = []
         labels = {}
         extra_headers = []  # separate list to append at the end.
         extra_labels = {}
-
+        _fields = request.GET.get("_fields") if request else None
+        requested_fields = (
+            [f for f in _fields.split(",") if not f.startswith("-")] if _fields else None
+        )
```

Comment:
```suggestion
        requested_fields = (
            [f for f in request.GET.get("_fields", "").split(",") if not f.startswith("-")] if request is not None else None
        )
```

Zou nog zo kunnen? 😬 ben je die `_fields` kwijt.

Reply:
oh ja..!